Objective-C/Tests/ReplicatorTest.m

```diff
@@ -1829,4 +1829,63 @@ - (void) testPushAndPullBigBodyDocument_SG {
     [self run: config errorCode: 0 errorDomain: nil];
 }
 
+
+- (void) testPushAndPullExpiredDocument_SG {
```

Comment:
This should be a functional test as it will not be run in an automated fashion.

Reply:
this will be run by QA. And not run during unit tests. So @pasin suggested to keep it, as long as it's not failing, QA can test this.